src/main/java/com/epam/processor/DataProcessor.java

```diff
@@ -51,7 +61,8 @@ public RoadAccident getAccidentByIndex7(String index){
      * @return
      */
     public Map<String, Long> getCountByRoadSurfaceCondition7(){
-        return null;
+        return roadAccidentList.stream()
+                .collect(Collectors.groupingBy(RoadAccident::getRoadSurfaceConditions, Collectors.counting()));
```

Comment:
Actually not. I was wondering what is needed here, so I click into the Collectors.groupingBy. And you will find some example there.